devices/src/virtio/block.rs

```diff
@@ -666,6 +670,8 @@ mod tests {
 
     use virtio::queue::tests::*;
 
+    const EPOLLIN: epoll::Events = epoll::Events::EPOLLIN;
```

Comment:
Why are you redefining this const?

Reply:
It's just defined as a shorthand, since `handle_event()` is called a whole bunch of times throughout the tests.

Reply:
Do this instead then:
```suggestion
    use epoll::Events::EPOLLIN;
```

Reply:
That wouldn't work, because `epoll::Events` is a bitflags struct, with some rust macro magic behind it.

Reply:
ack